plugins/gopacket/run.go

```diff
@@ -0,0 +1,247 @@
+// Copyright © 2021 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/signal"
+	"path"
+	"plugin"
+	"sort"
+	"strconv"
+	"syscall"
+
+	"github.com/op/go-logging"
+	log "github.com/sirupsen/logrus"
+	"github.com/up9inc/mizu/shared/logger"
+	"github.com/up9inc/mizu/tap"
+	"github.com/up9inc/mizu/tap/api"
+	"github.com/urfave/cli"
+
+	"github.com/apiclarity/apiclarity/plugins"
+	"github.com/apiclarity/apiclarity/plugins/api/client/client"
+	"github.com/apiclarity/apiclarity/plugins/api/client/client/operations"
+	"github.com/apiclarity/apiclarity/plugins/api/client/models"
+	"github.com/apiclarity/apiclarity/plugins/gopacket/config"
+	"github.com/apiclarity/apiclarity/plugins/gopacket/monitor"
+)
+
+type Agent struct {
+	podMonitor *monitor.PodMonitor
+	apiClient  *client.APIClarityPluginsTelemetriesAPI
+}
+
+func run(c *cli.Context) {
+	runConfig, err := config.LoadConfig()
+	if err != nil {
+		log.Errorf("Failed to load config: %v", err)
+		return
+	}
+
+	// load http plugin
+	extensions, err := loadExtensions()
+	if err != nil {
+		log.Errorf("Failed to load extensions: %v", err)
+		return
+	}
+	opts := &tap.TapOpts{
+		HostMode: true,
+	}
+
+	agent := &Agent{
+		apiClient: plugins.NewAPIClient(runConfig.UpstreamAddress),
+	}
+
+	// set mizu logger
+	logger.InitLoggerStderrOnly(logging.INFO)
```

Comment:
`logging.INFO` need to be configurable

Reply:
Not really this is just turning off the mizu logs. if you dont set it you get a LOT of noise in the logs (for each packet)

Reply:
we will probably want to be able to set it to `WARN` to avoid any info print, wouldn't we?